src/application/components/SyntaxHighlighter.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
```

Comment:
I think this can go?

Reply:
Good catch! Updated in [4b72ba3](https://github.com/apollographql/apollo-client-devtools/pull/994/commits/4b72ba3279c53f4754537a17858541a41efe17aa)